server/src/main/java/com/epam/aidial/core/server/data/Publication.java

```
@@ -54,6 +54,8 @@ public static class Resource {
          * Review resource url to review: files/review-bucket/folder/file.txt
          */
         String reviewUrl;
+
+        boolean publishCredentials;
```

Please add javadoc

---

added

---

Do we really need to return an error in case of no global credentials but the flag is on?

The code doesn't do in that way.